hazelcast/src/main/java/com/hazelcast/nio/ascii/TextReadHandler.java

```diff
@@ -62,7 +62,7 @@
 
     private static final Map<String, CommandParser> MAP_COMMAND_PARSERS = new HashMap<String, CommandParser>();
 
-    private static final int CAPACITY = 500;
+    private static final int CAPACITY = 5000;
```

Comment:
I thought the ByteBuffer was used for performance reasons. If you say dynamic, would you use a StringBuilder with a start capacity?